.travis_wait_build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# shell script that wait travis build result and report build state string
+#
+# QA_TOKEN  as arg1 : travis token to trigger qa build
+#
+# DEV NOTE:
+# - build takes between 3 and 4 minutes (but start could be delayed..)
+# - possible state: created|started|passed|failed
+# shell arg
+QA_TOKEN=$1
+QA_SLUG=geokrety%2Fgeokrety-website-qa
+QA_SLOG=geokrety/geokrety-website-qa
+
+if [ "${QA_TOKEN}" == "" ]; then
+  echo " X need QA_TOKEN as arg 1";
+  exit 1;
+fi
+
+ITERATION_RETRY_SLEEP=10
+ITERATION_MAX=30
+# 300 sec == 5 min max wait
+BUILD_STATE=""
+ITERATION=0
+
+function getBuildInfo() {
+    echo " * `date '+%Y-%m-%d %H:%M:%S'` [${ITERATION}] requesting ${QA_SLOG} builds";
+    BUILD_INFO=$(curl -s -X GET \
+       -H "Content-Type: application/json" \
+       -H "Accept: application/json" \
+       -H "Travis-API-Version: 3" \
+       -H "Authorization: token ${QA_TOKEN}" \
+       https://api.travis-ci.org/repo/${QA_SLUG}/builds?include=build.state\&include=build.id\&include=build.started_at\&branch.name=master\&sort_by=started_atdesc\&limit=1 )
```

Comment:
`branch.name=master` shouldn't it be `branch.name=${QA_BRANCH}`?

Reply:
we are talking about geokrety-website-qa branch here

I plant to maybe make it as env variable in the future
but for now we would like to tests using master branch of qa

Reply:
Ok, I didn't get it :)